lte/gateway/c/sctpd/CMakeLists.txt

```diff
@@ -13,6 +13,18 @@ cmake_minimum_required(VERSION 3.0.2)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
+get_cmake_property(vars CACHE_VARIABLES)
+foreach (var ${vars})
+  get_property(currentHelpString CACHE "${var}" PROPERTY HELPSTRING)
+  # message("${var} = [${${var}}]  --  ${currentHelpString}")
```

Comment:
do we want to keep this line?

Reply:
It is just easier to debug for posterity, but I don't have a strong preference lmk if you would still like me to remove it.

Reply:
If it's helpful to have it seems good to leave it in.